storm-dist/binary/src/main/assembly/binary.xml

```diff
@@ -30,38 +30,6 @@
             <useProjectArtifact>false</useProjectArtifact>
             <outputDirectory>lib</outputDirectory>
             <unpack>false</unpack>
-            <excludes>
```

Comment:
That was one thing what I didn't understand, and it was a sign that we didn't shade libraries properly. 
(Because assembly can see those packages...)
Do I understand correctly?